base-template/about-base-template.md

```diff
@@ -1,45 +1,63 @@
-# The Base Template
+# About Base Template
 
-## How do I create a new template?
+## Introduction
 
-Monotonectally engage orthogonal results vis-a-vis premier resources. Efficiently brand leveraged supply chains and interdependent best practices. Conveniently engineer resource-leveling opportunities without process-centric growth strategies. Appropriately implement intuitive materials without sticky scenarios. Rapidiously generate excellent channels before client-focused innovation.
+It's important to document repetitive or iterative processes that you encounter in your routine work. The best documentation uses a formal structure with specified standards, and adheres to predefined style guides. Doing this can sometimes be tedious, futile, and time consuming, so documentation can be random, ad hoc, and of inferior quality and a poor standard. If the documentation is poor quality to begin with, and is frequently edited by random different users, the quality and integrity can become even worse.
```

Comment:
This is selling to case for writing templates, which is a good case to make, but I think we are missing our target audience. I feel our target audience are template authors, who have already decided they need to create a template.
Our job now is to help them to make the template consistent with the best practices we are defining.
I'd suggest trim this section by 50% or more.